integration/go.mod

```diff
@@ -43,4 +43,5 @@ replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/crypto => ../crypto
 
-replace github.com/onflow/cadence => github.com/dapperlabs/cadence-internal v0.21.1-patch.2
+// v0.21.1-patch.2 seems missing
```

Comment:
so `github.com/dapperlabs/cadence-internal` does exist but is restricted. I can build this branch locally. 
So lets remove this comment.

Reply:
removed!